resources/mod_ssl.rb

```diff
@@ -60,7 +60,7 @@
     with_run_context :root do
       package 'mod_ssl' do
         notifies :run, 'execute[generate-module-list]', :immediately
-        only_if { platform_family?('rhel', 'fedora', 'suse', 'amazon') }
```

Comment:
?

Reply:
Should be just removing suse, suse doesn't have a mod_ssl ...